beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -884,6 +918,36 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         );
 
         let request_start_slot = Slot::from(req.start_slot);
+        // This variable may only change when the request_start_slot + req.count spans across the Fulu fork slot
+        let mut effective_count = req.count;
+
+        if let Some(fulu_epoch) = self.chain.spec.fulu_fork_epoch {
+            let fulu_start_slot = fulu_epoch.start_slot(T::EthSpec::slots_per_epoch());
+            let request_end_slot = request_start_slot + req.count - 1;
+
+            // If the request_start_slot is at or after a Fulu slot, return empty response
```

Comment:
// If the request_start_slot is at or after a Fulu slot, return an empty response